minet/youtube/client.py

```diff
@@ -70,6 +71,12 @@ def forge_videos_url(ids):
     return "%(base)s/videos?id=%(ids)s&part=snippet,statistics,contentDetails" % data
 
 
+def forge_channel_url(ids):
+    data = {"base": YOUTUBE_API_BASE_URL, "ids": ids}
+
+    return "%(base)s/channels?id=%(ids)s&part=snippet,statistics,contentDetails" % data
```

Comment:
Try adding at least `topicDetails` (and try other things listed in the docs: https://developers.google.com/youtube/v3/docs/channels/list). Ensure that there is no additional quota cost (the documentation does not state it does). This should take care of #524